geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/key/RenameNxNativeRedisAcceptanceTest.java

```diff
@@ -18,7 +18,7 @@
 
 import org.apache.geode.redis.NativeRedisClusterTestRule;
 
-public class RenamenxNativeRedisAcceptanceTest extends AbstractRenamenxIntegrationTest {
+public class RenameNxNativeRedisAcceptanceTest extends AbstractRenameNxIntegrationTest {
```

Comment:
Sorry to be nitpicky, but could this be "RenameNX" with a capital N and capital X? That's the naming pattern that has been used for other commands with NX versions.

Reply:
Ah yes, of course...